mecommerce/return-process
# esteban kocian feat(IntegrationTests): add first scenarios for integrations tests. mecommerce/feature/CXL-23

# Overview of the changes
Add modules needed for integration tests creation. Includes currently supported and un-supported scenarios:

Supported
- fullOneItemRefundWithTaxesWithShipping,
- fullTwoItemsSameSkuRefundWithTaxesWithShipping,
- fullTwoItemsDifferentSkuRefundWithTaxesWithShipping,
- partialTwoItemsSameSkuRefundWithTaxesWithShipping,
- partialTwoItemsDifferentSkuRefundWithTaxesWithShipping,
- fullMoreThanTwoItemsDifferentSkuRefundWithTaxesWithShipping,
- fullMoreThanTwoItemsSameSkuRefundWithTaxesWithShipping,

Not supported
- fullOneItemRefundWithDiscount,
- fullTwoItemsRefundWithDiscount,
- partialMoreThanTwoItemsRefund,
- fullMoreThanTwoItemsRefundWithDiscount,

### Jira Link
https://thirdlove.atlassian.net/browse/CXL-23

# Steps for testing
1. checkout the branch on your local env
2. Select the tests that you want to include to be run by setting the skip property present on the description cases files (i.e. test/integration/scenarios/test-cases-*)
2. run the script by `npm run test:integration`.
3. Also checkout a new branch with rc/* and check that the CI run the test and work properly.

# FYI / Please Review:
There is a complementary [document](https://thirdlove.atlassian.net/wiki/spaces/TLENG/pages/779289604) on Confluence about the covered Scenarios
